tests/test_utils.py

```diff
@@ -20,6 +20,47 @@
 assert infer_iri(coll.meta) == coll.meta.uri
 assert infer_iri(coll) == coll.uuid
 
+# We have no dependencies on pydantic, hence don't assume that it is installed
```

Comment:
A good starting point could be to use (or expand on) this datamodel: https://github.com/SINTEF/soft7/blob/main/s7/pydantic_models/soft7.py

Reply:
OK, I renamed `uri` to `identity` in the test. With that it seems that we are pretty much matching the soft7 datamodel. 

However, it is confusing and not very useful to have two different names for the same thing in soft7 and dlite. I have nothing in principle against updating dlite, but it would take several hours to change `uri` to `identity` allover dlite when also keeping backward compatibility. It is time I don't have. Great if someone would like to do it, otherwise I would suggest to update soft7 instead.